src/app/_models/addSittingRecords.model.ts

```diff
@@ -0,0 +1,16 @@
+export class SittingRecordsPostObj {
+    hmctsServiceCode!: string;
+    sittingDate!: Date;
+    epimsId!: string;
```

Comment:
shouldn't this be epimmsId?

Reply:
will be resolved as part of merge with IJPS-43